app/lib/audit/backend.dart

```diff
@@ -136,4 +139,56 @@ class AuditBackend {
     InvalidInputException.check(parsed != null, 'Unable to parse `before`.');
     return parsed!;
   }
+
+  /// Returns the entries from the last day.
+  ///
+  /// Keeps the [_cachedRecords] fields updated, and lists only the entries
+  /// up to the last query.
+  Future<List<AuditLogRecord>> getEntriesFromLastDay() async {
+    final now = clock.now().toUtc();
+    final cachedAge = now.difference(_cachedRecords.timestamp);
+
+    // fast track for requests within 5 seconds:
+    if (cachedAge.inSeconds < 5) {
+      return _cachedRecords.records;
+    }
```

Comment:
I don't mind by then we have concurrent query operations... that's perhaps a bit much :rofl: